spec/api/raw_api_spec.rb

```diff
@@ -3,23 +3,74 @@
 require 'socket'
 
 describe Uploadcare::Api do
-  before :each do
-    @api = Uploadcare::Api.new(CONFIG)
-  end
+  subject(:api) { Uploadcare::Api.new(CONFIG) }
 
   it "should initialize api" do
-    @api.should be_an_instance_of(Uploadcare::Api)
+    is_expected.to be_an_instance_of(Uploadcare::Api)
   end
 
   it 'should respond to request methods' do
-    @api.should respond_to :request
-    @api.should respond_to :get
-    @api.should respond_to :post
-    @api.should respond_to :put
-    @api.should respond_to :delete
+    is_expected.to respond_to :request
+    is_expected.to respond_to :get
+    is_expected.to respond_to :post
+    is_expected.to respond_to :put
+    is_expected.to respond_to :delete
   end
 
-  it 'should perform custom requests' do
-    expect { @api.request }.to_not raise_error
+  context 'when performing requests' do
+    subject(:request) { api.request }
+
+    it { is_expected.to be_a Hash }
+
+    context 'when checking keys' do
```

Comment:
`#request` method can be used for any api request, not only for fetching files (the default behavior), so in general it'll return some Hash and in my opinion there is no need to test it's contents. `it { is_expected.to be_a Hash }` will be enough.